src/components/ChatList/elements/ChannelImage.style.ts

```diff
@@ -1,7 +1,7 @@
 import {StyleSheet} from 'react-native';
 
 import dimen from '../../../utils/dimen';
-import {COLORS} from '../../../utils/theme';
+import {colors} from '../../../utils/colors';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement has been updated to use `colors` instead of `COLORS`. Ensure that the new `colors` module provides the same color definitions as the old `COLORS` module. If not, this could lead to unexpected changes in the app's appearance.
```diff
- import {COLORS} from '../../../utils/theme';
+ import {colors} from '../../../utils/colors';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->